api/src/main/java/org/openmrs/Person.java

```diff
@@ -325,6 +327,18 @@ public void setCauseOfDeath(Concept causeOfDeath) {
 		this.causeOfDeath = causeOfDeath;
 	}
 	
+	/**
+	 * @return cause of person's death non coded
+	 */
+	public String getCauseOfDeathNonCoded() {
+		return this.causeOfDeathNonCoded;
+	}
+	
+	/**
+	 * @return cause of person's death non coded
+	 */
+	public void setCauseOfDeathNonCoded(String causeOfDeathNonCoded) { this.causeOfDeathNonCoded = causeOfDeathNonCoded;}
```

Comment:
We prefer having the implementation on a separate line https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle